### DISCUSSION TITLE
Are there any format tokens to display day of month as an ordinal number, eg 1st ,2nd ,3rd ?

### DISCUSSION THREAD

**Discussion by cheekybuddha** [moment/luxon#1589] [Q&A] [answered]
If the requirement is to return a date in the format:
Saturday 27th January 2024

Or do I just have to use d or dd and try and perform some string-fu afterwards?

Apologies if this has been discussed before, couldn't find anything in a search.

**Comment by diesieben07** [comment, marked as answer]
Unfortunately there is no Browser API to do this kind of formatting.
See these issues:
https://github.com/moment/luxon/issues/1411
https://github.com/moment/luxon/issues/118
https://github.com/tc39/ecma402/issues/494

**Reply by cheekybuddha** [reply]
Many thanks for the clarification
